client/package.json

```diff
@@ -3,9 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
```

Comment:
I couldn't see these being used, do we need them?

Reply:
This is used by the Date and Time Pickers , they are from the mui/lab. before installing them the components weren't rendering